addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -338,6 +338,12 @@ function factory(dependencies) {
         isDoFocus: attr({
             default: false,
         }),
+        /**
+         * The chatter is inside .form_sheet_bg class
+         */
+        isInFormSheetBg: attr({
+            default: false,
+        }),
```

Comment:
This is not really a problem but it would have made more sense to add this field with a patch in enterprise. It's totally useless in community.